vtm.js

```diff
@@ -326,4 +476,122 @@ app.component('stat', {
     </div>`
 });
 
+/**
+  * displays inputs for name and age
+  * displays selects for clan, generation, faction
+  */
+app.component('character-info', {
+  props: ['bio', 'clans'],
+  
+  template: `
+    <div class="char-info">
+      <input 
+        class="heading1"
+        type="text"
+        v-model="bio.name" 
+        :placeholder="bio.defaultName"
+        />
+      <b>Age:</b>
+      <input
+        :style= "{width: 7 + 'ch'}"
+        v-model="bio.age"
+        type="number"
+        :placeholder = '123'/>
+      <b>Generation:</b>
+      <select 
+        v-model="bio.generation">
+        <option disabled value=null>-none-</option>
+        <option>15th</option>
+        <option>14th</option>
+        <option>13th</option>
+      </select>
+      <b>Clan:</b>
+      <select v-model="bio.clan">
+        <option disabled value="">Pick a Clan</option>
+        <option 
+          v-for="clan in clans"
+          :key = "clan.id"
+          :value="clan">
+          {{clan.id}}
+        </option>
+      </select>
+      <b>Faction:</b>
+      <select v-model="bio.faction">
+        <option disabled value="">Choose Your Coterie's Faction</option>
+        <option>Cammarilla</option>
```

Comment:
- None yet
- Anarch
- Camarilla
- Autarkis

Reply:
- Sabbat

Reply:
a Autarkis mas v podstate Anarch... aspon z pohladu Camarilly

Reply:
no ale kazdopadne by sa to malo previazat s datmi, data nemam, treba vymisliet ako budu dobre, a ja tym frakciam az tak nerozumiem tak asi nevidim nejaky okamzity uzitok, blood potency mi pride dolezitejsia

Reply:
Tak teda celu tu cast s faction zmazeme? :-)

Reply:
Z herneho pohladu to ma zmysel, ale cisla do toho ziadne nedostanes a ani z toho nic nevypocitas... Moze to byt kludne prazdny input.

Reply:
pre mna je srozumitelne:
- Camarilla
- Anarch - superi Camarily
- Sabbat - kaslu na ludskost

dalej rozdiel mezi:
- ziadna frakcia
- Autarkis

vnimam ako nerozhodnuty alebo rozhodnuty, ze nechce do prvych troch patrit

Reply:
a niesu Autarkis upiry bez frakcie? asi by som si mal k tym pravidlam zas sadnut a trochu ich nastudovat...
no frakcie tak nejak obmezuju klany lebo kamarila ma svoje klany, stejne tak anarchovia a sabbat, ja by som robil to ze podla frakcie by som disabloval klany, a ked neni frakcia tak si mozes vybrat klan ktory chces, takze stale mozes mat klan aky chces, ale ked chce PJ nejak omezit volbu klanov, tak potom si to hrac nemusi hlidat

Reply:
Ja som trosku cital vcera :-)

Autarkis je upir bez frakcie, ale imho pretoze zavrhuje Kamarilu, Anarchov aj Sabbat

ziadna frakcia vnimam ako, ze si frakciu zatial nevybral - ale to moze hrac nechat prazdne.

Imho tu zatial staci doplnit nazvy frakcii a pripadne disablovanie klanov urobit neskor